include/exadg/structure/user_interface/boundary_descriptor.h

```diff
@@ -70,6 +71,16 @@ struct BoundaryDescriptor
   // Neumann
   std::map<dealii::types::boundary_id, std::shared_ptr<dealii::Function<dim>>> neumann_bc;
 
+  // Robin boundary condition of the form
+  // + ( v_h, k * d_h + c * d/dt(d_h) +  p * N )
+  // or
+  // + ( v_h, k * N * (d_h . N) + c * N . (d/dt(d_h) . N) + p * N )
+  // using normal projections of displacement/velocity terms controlled via the
+  // std::array<bool, 2> for the displacement (index 0) and velocity terms (index 1)
+  // The std::array<double, 3> contains the parameters k (index 0), c (index 1) and p (index 2).
+  std::map<dealii::types::boundary_id, std::pair<std::array<bool, 2>, std::array<double, 3>>>
+    robin_k_c_p_param;
```

Comment:
This design makes the code very hard to read in my opinion, because in the implementation (`SpatialOperator` classes), one only sees `.first`, `.second`, etc., without concrete names.

Here, the data structure should look like
```c++
std::map<dealii::types::boundary_id, ExaDG::RobinParameters> robin_bc;
```

and inside `RobinParameters` variables like `.stiffness`, etc. with readable names. Instead of `robin_k_c_p_param` (where it is hard to guess the meaning), we should follow the naming `robin_bc` for consistency with other boundary types.